externals/metaphysics

```diff
@@ -1 +1 @@
-Subproject commit 849a8e2c1f7b7da61b06f152e5ea4ea34c769387
+Subproject commit 1541a0d42480bd95aee87898200a8ad88e086a45
```

Comment:
Changes: 
https://github.com/artsy/metaphysics/compare/849a8e2c1f7b7da61b06f152e5ea4ea34c769387...1541a0d42480bd95aee87898200a8ad88e086a45

I assume this got updated as a part of running `npm run sync-schema` - should be fine